pkg/scheduler/scheduler.go

```diff
@@ -150,6 +155,102 @@ type Config struct {
 	DisablePreemption bool
 }
 
+type schedulerOptions struct {
+	schedulerName                  string
+	hardPodAffinitySymmetricWeight int32
+	enableEquivalenceClassCache    bool
+	disablePreemption              bool
+	percentageOfNodesToScore       int32
+	bindTimeoutSeconds             int64
+	schedulerAlgorithmSource       kubeschedulerconfig.SchedulerAlgorithmSource
+}
+
+// Option is function for schedulerOptions to set value
```

Comment:
This comment is describing the implementation rather than the interface. How about, "Option configures a Scheduler"?